contracts/DCAHub/DCAHubParameters.sol

```diff
@@ -6,47 +6,46 @@ import '../interfaces/IDCAHub.sol';
 import '../libraries/TokenSorting.sol';
 
 abstract contract DCAHubParameters is IDCAHubParameters {
+  /// @notice Swap information about a specific pair
```

Comment:
Since we are not exposing the structs in the interface anymore, I moved them here